schemas/stsci.edu/asdf/0.1.0/core/history_entry.yaml

```diff
@@ -0,0 +1,28 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/0.1.0/core/history_entry"
+title: |
+  An entry in the file history.
+
+tag: "tag:stsci.edu:asdf/0.1.0/core/asdf"
+type: object
+properties:
+  description:
+    description: |
+      A description of the transformation performed.
+    type: string
+
+  time:
+    description: |
+      A timestamp for the operation.
+    type: string
+    format: date-time
+
+  software:
+    description: |
+      A description of the software that performed the operation.
```

Comment:
Presumably the amount of detail to include in this list would be up to the user.  For example a program that updates the ASDF file is free to write an entry into the history about itself.  But there's nothing stopping it from including however much detail it wants in the description field.  I agree there may be such a thing as "too much" here, but we don't have to decide that right now.